tests/ConversationTranslatorTests.ts

```diff
@@ -84,6 +86,401 @@ describe("conversation constructor tests", () => {
+
+        const config = sdk.SpeechTranslationConfig.fromSubscription("abc", "def");
+        config.speechRecognitionLanguage = "en-US";
+        if (endpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Host], endpointHost); }
+        if (speechEndpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+        const c = sdk.Conversation.createConversationAsync(config, undefined, ((error: any) => {
+            expect(error).toContain("401000");
+            errorMessage = error;
+        }));
+        objsToClose.push(c);
+
+        WaitForCondition(() => (errorMessage !== undefined), done);
+
+    }, 20000);
+
+    test("Start Conversation, join as host and mute participants", (done: jest.DoneCallback) => {
+
+        // tslint:disable-next-line:no-console
+        console.info("Start Conversation, join as host, mute participants");
+
+        let commandCount: number = 0;
+        let participantsCount: number = 0;
+        let isMuted: boolean = true;
+
+        // start a conversation
+        const config = sdk.SpeechTranslationConfig.fromSubscription(Settings.SpeechSubscriptionKey, Settings.SpeechRegion);
+        if (endpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Host], endpointHost); }
+        if (speechEndpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+        const c: sdk.Conversation = sdk.Conversation.createConversationAsync(config, (() => {
+            const ct: sdk.ConversationTranslator = new sdk.ConversationTranslator();
+            if (endpointHost !== "") { ct.properties.setProperty(sdk.PropertyId.ConversationTranslator_Host, endpointHost); }
+            if (speechEndpointHost !== "") { ct.properties.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+            ct.canceled = ((s: sdk.ConversationTranslator, e: sdk.ConversationTranslationCanceledEventArgs) => {
+                done();
+            });
+            ct.participantsChanged = ((s: sdk.ConversationTranslator, e: sdk.ConversationParticipantsChangedEventArgs) => {
+                if (e.reason === sdk.ParticipantChangedReason.JoinedConversation) {
+                    participantsCount++;
+                    if (participantsCount === 1) {
+                        joinParticipant(c.conversationId);
+                    } else {
+                        c.muteAllParticipantsAsync();
+                    }
+                } else if (e.reason === sdk.ParticipantChangedReason.Updated) {
+                    commandCount++;
+                    if (e.participants.length > 0) {
+                        // only the participants should be updated by a mute all command
+                        expect(e.participants[0].isMuted).toEqual(isMuted);
+                        isMuted = !isMuted;
```

Comment:
>isMuted = !isMuted; [](start = 24, length = 19)

Why reverse this?

Reply:
It was probably a leftover from an older version of this test. Will remove it.

Reply:
I simplified this test.